capstone/src/main/resources/static/user.js

```diff
@@ -167,29 +167,22 @@ function getIndividualEventsOrOrganizations(isEvent) {
 }
 
 /* Function to create the individual organization display divs*/
-function createSavedOrgElement(data, deleteAllowed, displayButton) {
- const divElement = document.createElement('div');
- divElement.setAttribute("class", "item-container general-container");
-
- const aElementName = document.createElement('a');
- aElementName.setAttribute("class", "public-org-name");
- aElementName.innerText = data.name;
- aElementName.setAttribute("href", "publicprofile.html#" + data.datastoreId);
- divElement.appendChild(aElementName);
-
- const h5ElementEmail = document.createElement('h5');
- h5ElementEmail.innerText = data.email;
- divElement.appendChild(h5ElementEmail);
-
- const h5ElementBio = document.createElement('h5');
- h5ElementBio.innerText = data.description;
- divElement.appendChild(h5ElementBio);
-
- if (displayButton) {
-   const form = deleteAllowed ? createDeleteButton(data) : createSaveButton(data);
-   divElement.appendChild(form);
- }
- return divElement;
+function createSavedOrgElement(orgListElement, data, deleteAllowed, displayButton) {
+  const orgElement = createElement(orgListElement, 'li', '');
+  orgElement.className = 'event';
+
+  // Click to redirect to public organization page
+  orgElement.addEventListener('click', () => {
+    window.location="publicprofile.html#" + data.datastoreId;
```

Comment:
For future reference: It's more common to use the query string (`?field=value`) rather than the anchor (`#section`) to specify dynamic values. The latter is largely a frontend-only concept, while the former is easily read on the backend. I know you made the anchor tags work on the backend, too, but it's not the way they are typically used.